pyvista/themes.py

```diff
@@ -2198,6 +2204,50 @@ def use_ipyvtk(self, value):  # pragma: no cover
         else:
             self.jupyter_backend = 'static'
 
+    @property
+    def split_sharp_edges(self) -> bool:
+        """Set or return splitting sharp edges.
+
+        See :ref:`shading_example` for an example showing split sharp edges.
+
+        Examples
+        --------
+        Enable the splitting of sharp edges globally.
+
+        >>> import pyvista
+        >>> pyvista.global_theme.split_sharp_edges = True  # doctest:+SKIP
+        >>> pyvista.global_theme.split_sharp_edges  # doctest:+SKIP
+        True
+
+        """
+        return self._split_sharp_edges
+
+    @split_sharp_edges.setter
+    def split_sharp_edges(self, value: bool):
+        self._split_sharp_edges = value
+
+    @property
+    def sharp_edges_feature_angle(self) -> float:
+        """Set or return the angle of the sharp edges feature angle.
+
+        See :ref:`shading_example` for an example showing split sharp edges.
+
+        Examples
+        --------
+        Change the sharp edges feature angle to 45 degrees.
+
+        >>> import pyvista
+        >>> pyvista.global_theme.sharp_edges_feature_angle = 45.0  # doctest:+SKIP
+        >>> pyvista.global_theme.sharp_edges_feature_angle  # doctest:+SKIP
+        45.0
```

Comment:
```suggestion
        >>> pyvista.global_theme.sharp_edges_feature_angle = 45.0
        >>> pyvista.global_theme.sharp_edges_feature_angle
        45.0
```

Reply:
These have to be skipped. Initially when it wasn't, it was modifying the global them and messing up some of our doc tests.

Reply:
That's too bad...

Reply:
It's something to consider, but the issue you'd might run into is if an example fails (for some reason), we would be stuck in an invalid state and other examples might fail as well.

One way to work around this would be to reset the global theme at the end of every test via a fixture, similar to our `autoclose_plotters` fixture. In fact, might be the best solution, since we won't run the risk of accidentally modifying the state of the global theme.

This will end up being quite a "bonus", so I'll do this in a follow-up since I would prefer keeping "one PR, one feature/bug".